init_env

```diff
@@ -62,6 +62,13 @@ else
 	echo "Please, source this configuration from a terminal running BASH."
 fi
 
+if [[ -d $LISA_HOME/.git/hooks ]]; then
```

Comment:
init_env is only the toplevel wrapper with shell-specific setup, everything else is done in shell/lisa_shell.
You can add a function there:
```
function _lisa-install-git-hooks {
     # works even with worktrees
     local hooks="$(git -C "$LISA_HOME" rev-parse --git-common-dir)/hooks"
     # Avoid overriding user hooks
     [[ ! -e "$hooks/post-merge" ]] && cp "$LISA_HOME/tools/post-merge" "$hooks/post-merge"
     [[ ! -e "$hooks/post-checkout" ]] && cp "$LISA_HOME/tools/post-checkout" "$hooks/post-checkout"
}
```
and call it at the end of `lisa-install`. You will also need to update:
https://lisa-linux-integrated-system-analysis.readthedocs.io/en/master/setup.html#updating

Reply:
That's a lot better, yes! Thanks

Reply:
`[[ ! -e "$hooks/post-merge" ]]` check won't be good enough. we will fail to update it after the first install by the way